README.md

```diff
@@ -873,7 +864,7 @@ package.json:
 ```json
 {
   "scripts": {
-      "build": "cross-env NODE_ENV=production webpack --mode production --progress"
+      "build": "cross-env NODE_ENV=production vite build --mode=app"
     }
 }
 ```
```

Comment:
suggestion: I would recommend to remove this whole "Error on Windows machine" section, since it most likely is not applicable to Vite anyway.